.circleci/config.yml

```diff
@@ -226,7 +211,7 @@ commands:
             $msys2 pacman --sync --noconfirm --needed mingw-w64-x86_64-toolchain mingw-w64-x86_64-libtool unzip autoconf automake
 
   generic_build:
-    description: Run basic build and store in workspace for re-use by different architectures
+    description: Run basic build
```

Comment:
Can we add a comment explaining that the build step is very short, about 2 minutes, and that makes it faster to run as a step in testing jobs, rather than as a job on its own, persisting its output.

Reply:
@jannotti https://github.com/algorand/go-algorand/pull/4426/commits/cbb44866801b572235be78fe183fe41643ba3907 adds a comment. Can you resolve if it looks good to you?